styles.js

```diff
@@ -2,68 +2,100 @@ import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
 
-@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@200..800&family=Playfair+Display:ital,wght@0,900;1,900&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@200..800&family=Playfair+Display:ital,wght@0,900;1,900&display=swap');
+:root {
```

Comment:
This is a great start for a design system! Consider expanding it with additional variables for a more unified and reusable design approach. _Please take a look at my example in the PR Summary._